lisa/transformers/hyperv_preparation.py

```diff
@@ -0,0 +1,63 @@
+from dataclasses import dataclass
+from typing import Any, Dict, List, Type
+
+from dataclasses_json import dataclass_json
+
+from lisa import schema
+from lisa.node import quick_connect
+from lisa.tools import PowerShell
+from lisa.transformers.deployment_transformer import (
+    DeploymentTransformer,
+    DeploymentTransformerSchema,
+)
+
+
+@dataclass_json
+@dataclass
+class HyperVPreparationTransformerSchema(DeploymentTransformerSchema):
+    pass
+
+
+class HyperVPreparationTransformer(DeploymentTransformer):
+    """
+    This Transformer configures Windows Azure VM as a Hyper-V environment.
+    """
+
+    @classmethod
+    def type_name(cls) -> str:
+        return "hyperv_preparation"
+
+    @classmethod
+    def type_schema(cls) -> Type[schema.TypedSchema]:
+        return HyperVPreparationTransformerSchema
```

Comment:
It can return `DeploymentTransformerSchema`, no need another schema without new fields.

Reply:
Modified as suggested.